README.MARCJOSEPH.md

```diff
@@ -0,0 +1,31 @@
+# Car Rules 
+general Usage Rules
+
+1.	The vehicle can only go to church, school, and restaurants. All other destinations require permission.
```

Comment:
add (permission is given with a pull request to a file "REQUESTLOG.MARCJOSEPH.md")

- <date> <request>

Reply:
update the "restaurants" to be "restaurants within 15 mile radius of home"